postgresql.service.in

```diff
@@ -7,6 +7,8 @@
 [Unit]
 Description=PostgreSQL database server
 After=network.target
+# We should start postgresql service after network is up (rhbz#2127534)
+After=network-online.target
```

Comment:
Note there are two service files. (one with `@`).

Reply:
Ah, but there's only one source though (the one edited).  Please scratch my previous comment.

Reply:
Note though that the PGRPMS project replaced After=network.target with After=network-online.target.  The current proposal is adding a new dependency.

Reply:
Yes, I've synced it with the upstream